util/platform/Sockaddr.c

```diff
@@ -13,6 +13,7 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 #include "util/events/libuv/UvWrapper.h"
```

Comment:
I believe you can now remove UvWrapper.h

Reply:
Oops no, it fails miserably with 20 errors without that 🤔 

```
  Error: gcc -c -x cpp-output -o /Users/rogerio.sampaio/Documents/work/cjdns/target/release/build/cjdns_sys-2007d3eb81e9059e/out/util_platform_Sockaddr_c.o -std=c99 -Wall -Wextra -Wno-pointer-sign -Wmissing-prototypes -pedantic -D darwin=1 -Wno-unused-parameter -fomit-frame-pointer -ffunction-sections -fdata-sections -D Log_DEBUG -g -D NumberCompress_TYPE=v3x5x8 -D Identity_CHECK=1 -D Allocator_USE_CANARIES=1 -D PARANOIA=1 -march=native -DHAS_ETH_INTERFACE=1 -fPIE -Wno-invalid-pp-token -Wno-dollar-in-identifier-extension -Wno-newline-eof -Wno-unused-value -Wno-tautological-compare -Wno-error -O3 -D_FORTIFY_SOURCE=2 -fno-stack-protector -fstack-protector-all -Wstack-protector -D CJD_PACKAGE_VERSION="cjdns-v21.2-12-gaba96c54-dirty" /Users/rogerio.sampaio/Documents/work/cjdns/target/release/build/cjdns_sys-2007d3eb81e9059e/out/util_platform_Sockaddr_c.i

  util/platform/Sockaddr.c:33:29: error: field has incomplete type 'struct sockaddr_storage'
      struct sockaddr_storage ss;
                              ^
  util/platform/Sockaddr.c:33:12: note: forward declaration of 'struct sockaddr_storage'
      struct sockaddr_storage ss;
             ^
  util/platform/Sockaddr.c:39:24: error: field has incomplete type 'struct sockaddr_in'
      struct sockaddr_in si;
                         ^
  util/platform/Sockaddr.c:39:12: note: forward declaration of 'struct sockaddr_in'
      struct sockaddr_in si;
             ^
  util/platform/Sockaddr.c:44:25: error: field has incomplete type 'struct sockaddr_in6'
      struct sockaddr_in6 si;
                          ^
  util/platform/Sockaddr.c:44:12: note: forward declaration of 'struct sockaddr_in6'
      struct sockaddr_in6 si;
             ^
  util/platform/Sockaddr.c:51:27: error: use of undeclared identifier 'AF_INET'
              .sin_family = AF_INET,
                            ^
  util/platform/Sockaddr.c:59:27: error: use of undeclared identifier 'AF_INET'
              .sin_family = AF_INET,
                            ^
  util/platform/Sockaddr.c:67:28: error: use of undeclared identifier 'AF_INET6'
              .sin6_family = AF_INET6,
                             ^
  util/platform/Sockaddr.c:139:81: error: incomplete definition of type 'struct sockaddr_in6'
          if (Rffi_inet_pton(1, (char*) ((buff[0] == '[') ? &buff[1] : buff), &in6->sin6_addr)) {
                                                                               ~~~^
  util/platform/Sockaddr.c:44:12: note: forward declaration of 'struct sockaddr_in6'
      struct sockaddr_in6 si;
             ^
  util/platform/Sockaddr.c:142:29: error: invalid application of 'sizeof' to an incomplete type 'struct sockaddr_in6'
          out->addr.addrLen = sizeof(struct sockaddr_in6) + 8;
                              ^     ~~~~~~~~~~~~~~~~~~~~~
  util/platform/Sockaddr.c:44:12: note: forward declaration of 'struct sockaddr_in6'
      struct sockaddr_in6 si;
             ^
  util/platform/Sockaddr.c:143:12: error: incomplete definition of type 'struct sockaddr_in6'
          in6->sin6_port = ((uint16_t)(__extension__ ({ uint16_t _x = (((uint16_t)(port))); _x >> 8 | _x << 8; })));
          ~~~^
  util/platform/Sockaddr.c:44:12: note: forward declaration of 'struct sockaddr_in6'
      struct sockaddr_in6 si;
             ^
  util/platform/Sockaddr.c:144:12: error: incomplete definition of type 'struct sockaddr_in6'
          in6->sin6_family = AF_INET6;
          ~~~^
  util/platform/Sockaddr.c:44:12: note: forward declaration of 'struct sockaddr_in6'
      struct sockaddr_in6 si;
             ^
  util/platform/Sockaddr.c:144:28: error: use of undeclared identifier 'AF_INET6'
          in6->sin6_family = AF_INET6;
                             ^
  util/platform/Sockaddr.c:147:48: error: incomplete definition of type 'struct sockaddr_in'
          if (Rffi_inet_pton(0, (char*) buff, &in->sin_addr)) {
                                               ~~^
  util/platform/Sockaddr.c:39:12: note: forward declaration of 'struct sockaddr_in'
      struct sockaddr_in si;
             ^
  util/platform/Sockaddr.c:150:29: error: invalid application of 'sizeof' to an incomplete type 'struct sockaddr_in'
          out->addr.addrLen = sizeof(struct sockaddr_in) + 8;
                              ^     ~~~~~~~~~~~~~~~~~~~~
  util/platform/Sockaddr.c:39:12: note: forward declaration of 'struct sockaddr_in'
      struct sockaddr_in si;
             ^
  util/platform/Sockaddr.c:151:11: error: incomplete definition of type 'struct sockaddr_in'
          in->sin_port = ((uint16_t)(__extension__ ({ uint16_t _x = (((uint16_t)(port))); _x >> 8 | _x << 8; })));
          ~~^
  util/platform/Sockaddr.c:39:12: note: forward declaration of 'struct sockaddr_in'
      struct sockaddr_in si;
             ^
  util/platform/Sockaddr.c:152:11: error: incomplete definition of type 'struct sockaddr_in'
          in->sin_family = AF_INET;
          ~~^
  util/platform/Sockaddr.c:39:12: note: forward declaration of 'struct sockaddr_in'
      struct sockaddr_in si;
             ^
  util/platform/Sockaddr.c:152:26: error: use of undeclared identifier 'AF_INET'
          in->sin_family = AF_INET;
                           ^
  util/platform/Sockaddr.c:182:14: error: use of undeclared identifier 'AF_INET'
          case AF_INET:
               ^
  util/platform/Sockaddr.c:183:56: error: incomplete definition of type 'struct sockaddr_in'
              inAddr = &((struct sockaddr_in*) &addr->ss)->sin_addr;
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
  util/platform/Sockaddr.c:39:12: note: forward declaration of 'struct sockaddr_in'
      struct sockaddr_in si;
             ^
  util/platform/Sockaddr.c:184:108: error: incomplete definition of type 'struct sockaddr_in'
              port = ((uint16_t)(__extension__ ({ uint16_t _x = (((uint16_t)(((struct sockaddr_in*)&addr->ss)->sin_port))); _x >> 8 | _x << 8; })));
                                                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
  util/platform/Sockaddr.c:39:12: note: forward declaration of 'struct sockaddr_in'
      struct sockaddr_in si;
             ^
  fatal error: too many errors emitted, stopping now [-ferror-limit=]
  20 errors generated.

      at error (/Users/rogerio.sampaio/Documents/work/cjdns/node_build/builder.js:115:15)
      at /Users/rogerio.sampaio/Documents/work/cjdns/node_build/builder.js:196:29
      at /Users/rogerio.sampaio/Documents/work/cjdns/node_build/builder.js:161:17
      at ChildProcess.<anonymous> (/Users/rogerio.sampaio/Documents/work/cjdns/node_modules/saferphore/index.js:22:34)
      at ChildProcess.emit (node:events:390:28)
      at maybeClose (node:internal/child_process:1064:16)
      at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
  Error: Failed to build cjdns
```

Reply:
Done!